pkg/remote/azurerm_network_scanner_test.go

```diff
@@ -52,23 +52,25 @@ func TestAzurermVirtualNetwork(t *testing.T) {
 				repository.On("ListAllVirtualNetworks").Return([]*armnetwork.VirtualNetwork{
 					{
 						Resource: armnetwork.Resource{
-							ID: to.StringPtr("network1"),
+							ID:   to.StringPtr("/subscriptions/bbc8a244-64b6-4f28-9d73-18869ab6a361/resourceGroups/driftctl-tests/providers/Microsoft.Network/virtualNetworks/network1"),
```

Comment:
Why changing the ID ? It was a voluntary not to put something complicated here since it is only for testing purpose.

Reply:
it was just about using realistic data but I can revert this change if you want